README.lhs

```diff
@@ -109,8 +109,10 @@ When using `envSettings`, the following variables will be used:
 - `FAKTORY_PROVIDER`: the name of another environment variable where the
   connection string can be found. Defaults to `FAKTORY_URL`.
 - `FAKTORY_URL` (or whatever you named in `FAKTORY_PROVIDER`): connection string
-  to the Faktory server. Format is `tcp(+tls)://(:password@)host:port`. Defaults
-  to `tcp://localhost:4719`.
+  to the Faktory server. Format is
+  `tcp(+tls)://(:password@)host:port(/namespace)`. Defaults to
+  `tcp://localhost:4719`. `namespace` is prependend to queue names on job
+  submission.
```

Comment:
Nit, but probably good to state "and/or consumption" too?

**EDIT**: This is no longer a nit -- I can't see where you namespace the queues on consumption too. Am I missing something?

Reply:
Ah you're right, we should do that too.

I was thinking "we can do that without any code changes by just passing in FAKTORY_QUEUE=`{Env}Low`", but hell yeah consistency. Will fix.